yaml-cpp-config.cmake.in

```diff
@@ -11,12 +11,12 @@ set_and_check(YAML_CPP_INCLUDE_DIR "@PACKAGE_CMAKE_INSTALL_INCLUDEDIR@")
 set_and_check(YAML_CPP_LIBRARY_DIR "@PACKAGE_CMAKE_INSTALL_LIBDIR@")
 
 # Are we building shared libraries?
-set(YAML_CPP_SHARED_LIBS_BUILT "@PACKAGE_YAML_BUILD_SHARED_LIBS@")
+set(YAML_CPP_SHARED_LIBS_BUILT @YAML_BUILD_SHARED_LIBS@)
 
 # Our library dependencies (contains definitions for IMPORTED targets)
-include(@PACKAGE_CONFIG_EXPORT_DIR@/yaml-cpp-targets.cmake)
+include("${CMAKE_CURRENT_LIST_DIR}/yaml-cpp-targets.cmake")
```

Comment:
Would you mind adding something along the lines of this code to the config file?
```
# Add an alias for backward compatibility with <= v0.7.
add_library(yaml-cpp ALIAS yaml-cpp::yaml-cpp)
```

There is at least [one instance where client code uses](https://github.com/AcademySoftwareFoundation/OpenColorIO/blob/e670cd98d794e83cd5a9d079f8979d9204466128/share/cmake/modules/Findyaml-cpp.cmake#L37) the `yaml-cpp` target directly, instead of using `YAML_CPP_LIBRARIES`, and adding this alias makes the transition a lot easier.